tests/waku_example.py

```diff
@@ -4,7 +4,12 @@
 
 def handle_event(ret, msg):
   if ret == 0: ## RET_OK
-    print("Ok event received: %s" % msg)
+    msgPlain = ""
+    try:
+      msgPlain = msg.decode('utf-8')
+    except:
+      msgPlain = "[binary content]"
```

Comment:
I wonder if we could append somehow the exception detail

Reply:
Well, the exception is something like "Failed to decode because...", but it is not really useful because we know what the problem is - the msg is in binary format, not plaintext utf-8, so I just wanted to give that note to the user, but the main point is that the message got delivered